Description: remove reference to obsolete syslog.target
 Syslog is socket-activated, so no need to declare a dependency on it
Author: Damyan Ivanov <dmn@debian.org>

--- a/builds/install/arch-specific/linux/firebird-superserver.service.in
+++ b/builds/install/arch-specific/linux/firebird-superserver.service.in
@@ -1,6 +1,6 @@
 [Unit]
 Description=Firebird Database Server ( SuperServer )
-After=syslog.target network.target
+After=network.target
 Conflicts=firebird-classic.socket
 
 [Service]
